week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -68,7 +68,11 @@ function checkCodeIsThere(stringText) {
   
   Hint: Use the corresponding array method to split the array.
 */
-function getTransportModes() { }
+function getTransportModes(arr) {
+  // arr.shift();
+
+  return arr.slice(1);
```

Comment:
using slice is a good call, but remember you want only the first item 
the way you wrote it, will take every item start on 1